qemu/tests/cfg/multi_host.cfg

```diff
@@ -315,6 +315,41 @@
                             attempts = 100
                             attempts_timeout = 1
 
+                - virtio_blk_data_plane:
+                    only Linux
+                    only Host_RHEL.m7
+                    only virtio_blk
+                    type = migration_multi_host
+                    preprocess_env = yes
+                    iothreads = iothread0
+                    blk_extra_params_image1 = "iothread=${iothreads}"
+                    drive_werror_image1 = stop
+                    drive_rerror_image1 = stop
+                    migration_bg_command = "fio -filename /dev/vda --runtime 600 --rw=randread -name=randread0 > /dev/null &"
+                    migration_bg_check_command = "pgrep fio"
+                    migration_bg_kill_command = "pkill -9 fio"
+
+                - with_module_reload:
+                    only Linux
+                    type = migration_multi_host
+                    preprocess_env = yes
+                    serial_login = yes
+                    not_login_after_mig = yes
+                    post_sub_test = "file_transfer"
+                    filesize = 100
+                    transfer_timeout = 600
+                    virtio_net:
+                        module_name = virtio_net
+                    e1000:
+                        module_name = e1000
+                    rtl8139:
+                        module_name = 8139cp
+                    migration_bg_command = "cd /tmp;"
+                    migration_bg_command += " echo "while true; do rmmod ${module_name}; modprobe ${module_name}; sleep 3; done" > test.sh;"
```

Comment:
@xutian could you review this patch,
The code will be weak, if just send the bg cmd without any handling, we don't if the script is running, what happen in guest while test run in guest.

Thanks